idaes/models_extra/power_generation/costing/power_plant_capcost.py

```diff
@@ -1051,14 +1219,26 @@ def get_ASU_cost(self, scaled_param):
             fs = self.parent_block()
 
         # build flowsheet level parameters CE_index = year
-        if not hasattr(fs, "costing"):
-            fs.get_costing(year="2017")
+        # if not hasattr(fs, "costing"):
+        #     fs.get_costing(year="2017")
 
-        CE_index = fs.costing.CE_index
+        # CE_index = fs.costing.CE_index
```

Comment:
delete

Reply:
Would you like all of the commented code deleted?